.gitignore

```
@@ -128,3 +128,6 @@ dmypy.json
 # Pyre type checker
 .pyre/
 pyrightconfig.json
+
+settings.json
+!.vscode/settings.json
```

```suggestion
/settings.json
```
Use `/settings.json` to only match the app settings file instead